.github/workflow-templates/readme-template.md

```diff
@@ -0,0 +1,16 @@
+# <program> container

```

Comment:
oh yeah. Didn't mean to include this, but it can stay in.

Reply:
sounds good!

Reply:
sounds good!